server/models/escalationPolicy.js

```diff
@@ -9,16 +9,24 @@ import mongoose from 'mongoose';
  */
 
 const EscalationPolicySchema = new mongoose.Schema({
-  rotationInterval: Number,
+  rotationInterval: {
```

Comment:
I'd like to see some kind of documentation on these intervals, so we know the units. 

I'd prefer something like changing the variable name to `rotationIntervalInDays` or something similar (so that you don't have to search super far into the code to figure out the units), but I'd also be content with a comment in this file that outlines the units.